### DISCUSSION TITLE
Feedback for Translation of Brazilian Portuguese Discussions

### DISCUSSION THREAD

**Discussion by bigzoo** [community/community#4374] [Discussions]
Hello! :wave:
This is a discussion for gathering user feedback on the beta release of Brazilian Portuguese translations for Discussions.

To learn more about this feature and how to use it, see the [changelog post](https://github.blog/changelog/2021-07-19-machine-translation-for-discussions-content/).

Please use this discussion to share your experience using the new feature, and any ideas you have on how the experience could be improved.

**Comment by JNylson** [comment]
yes, very good

**Reply by ShyCode-analyst** [reply]
yes, very good

**Reply by regidf12** [reply]
I like

**Reply by Putt0** [reply]
Cool

**Reply by PothpothBR** [reply]
Helps a lot

**Reply by Munzambi-Miguel** [reply]
Realmente essa tradução foi boas e aproveitavel

**Reply by sueliitj** [reply]
Yes

**Reply by LuanSchneider** [reply]
like

**Reply by gabisdias** [reply]
perfect!

**Reply by Crystoppher** [reply]
Thanks! 🎉 👌🏾

**Comment by Gabb-c** [comment]
LGTM

**Comment by ceohockey60** [comment]
Hi @JNylson @Gabb-c , thank you for your feedback. Do you think this type of feature is useful for Brazilian developers? Should this feature exist in other parts of GitHub, like Issues or PRs as well?

**Reply by JNylson** [reply]
yes I would like

**Reply by Gabb-c** [reply]
yes, me too

**Reply by mribeirodantas** [reply]
I think Brazilian developers can definitely benefit from this feature, @ceohockey60. I think it'd be nice to have both versions showing up, as you have on other websites. The translation is not so good sometimes, so having both versions displayed can make it easier for the user to understand.

**Comment by veleirocica** [comment]
Very good.

**Comment by ghost** [comment]
Very good.

**Comment by michaelbs09** [comment]
Very good

**Comment by fgsoftware1** [comment]
very good

**Comment by CelsoMMiranda** [comment]
Muito boa!

**Comment by tiagofrancafernandes** [comment]
Boa! Good

**Comment by marcussacana** [comment]
I lilked, I supose is using bing translator as base, isn't the best MTL service but isn't bad as well :) good job, github team.

**Reply by RoinujNosde** [reply]
Bing and github are owned by Microsoft, so that's expected.

**Reply by r0b14** [reply]
@RoinujNosde in fact, cooperation between both tools is to be expected.

**Comment by marcussacana** [comment]
I think should be implemented even in the readme's, otherwise will look like a 'half implemented' feature, maybe as well, in the user settings allow us to enable auto translation to some parts of the github, like the discussion's, readme's, wiki's... all independently, being able to turn off or not in certain parts the automatic translation.

I personally supose the advantage of a embbed translation feature is how will be possible to we translate only the text while we skip code blocks, since some translators just translate anything including the source code :)

**Reply by zeke** [reply]
> I think should be implemented even in the readme's

Great idea. I would love to see that too. 

> skip code blocks

The current translation feature already does skip code blocks. 👍🏼

**Reply by marcussacana** [reply]
Yeah, I just wanted quote, since is a cool feature that make difference, if  we compare with external web translators like google translate, they aren't smart enought to skip the code.

**Reply by Carlos-Levi** [reply]
boa ideia !!

**Reply by r0b14** [reply]
I thought this idea was fantastic. I'd thought of something similar, as I'm always trying to figure out how we would understand concepts and vocabulary that are already rooted in English.

**Comment by ArithmosPrtgn** [comment]
It's great!

**Comment by samuel-lujan** [comment]
It's amazing! something that could get better it´is also translated the title. This could be useful

**Comment by voidcoefficient** [comment]
surprisingly good

**Comment by ceohockey60** [comment]
Thank you all for the feedback. Please continue to give us your feedback, share with other Developers to try this out, as we mature this feature!

**Comment by tiagonevestia** [comment]
It's great!

**Comment by mdsdias** [comment]
I'm loving it. Thank you for including us more.

**Comment by omalisson** [comment]
Perfeito!

**Comment by ceohockey60** [comment]
Thank you all for your feedback. Looks like the machine translation quality is quite good.

What do you think about adding this feature to other parts of GitHub, like Issues, PR's, etc.?

**Comment by Alfraro** [comment]
It was very good.

**Comment by otoniel19** [comment]
yes it was very good

**Comment by solundev** [comment]
i find it more than usefull ^-^

**Comment by maikvortx** [comment]
i enjoy this function!

**Comment by BaziotaBeans** [comment]
Amazing!

**Comment by gdcmarinho** [comment]
To be honest, sometimes some words get a little meaningless. It seems like they literally translate what the words mean and don't take the context of the sentence.

**Reply by gdcmarinho** [reply]
It's a good feature, but I think that could be better.

**Comment by JoaoEmanuell** [comment]
Excelente feature.
I would really like it was also implemented for issues and pull requests as well.

**Comment by pwccaa** [comment]
Awesome feature, but imo it shouldnt translate terms like "Python".

**Comment by scush989898** [comment]
Amazing

**Comment by LearXD** [comment]
Good!!

**Comment by lucioerlan** [comment]
great XD

**Comment by felipefetec** [comment]
Very good, show (Muito bom, show)

**Comment by vinicioslc** [comment]
Pretty Nice !!

**Comment by dresende** [comment]
It would be nice to say "Traduzir para portugês do brasil" so portuguese people from europe know it's not portuguese. There are many words that are different and sound awful for us when reading.

These differences are not subtle as is in the case of us english or uk english.

**Comment by Birdza87** [comment]
Hello

**Reply by Birdza87** [reply]
> Hello

**Comment by AdriansFernandes** [comment]
Very useful for us Brazilians, especially for those who don't understand or are starting to use GitHub, I haven't seen how far this support will go, but it would be nice to go for everything, a plus would be if GitHub was translated into Brazilian Portuguese as well.

**Comment by 0spura** [comment]
Sometimes the translation does not put some markdowns, like `test`, in the original comment order. Other than that, it's pretty good.

**Comment by BernardoGP4504** [comment]
Nice feature but sometimes the translation is wrong or it does not translate certain words but, overall a nice feature. Though i feel like it should be implemented in readme's, discussions, issues and wikis and, that you could toggle a setting so it automatically translates something (issues, etc.)

**Comment by vitordevsp** [comment]
Muito bom!

**Comment by mendoncart** [comment]
Yes, it's working surprisingly well!

**Comment by ceolinwill** [comment]
@bigzoo @ceohockey60 Is there a way to disable this feature? I tried setting _English_ as my preferred language in the settings but the "translate to Portuguese" option is still showing up on every comment. I think it's a good feature but it can be a bit distracting for those who speak English.

**Reply by bigzoo** [reply]
We're both no-longer at GH but I think the only way to 'disable' it would be changing the default lang of your browser to English.

**Reply by jrom99** [reply]
Is there no other way? It's really distracting having one button per comment, but I don't want to be forced to change my entire browser's language just to improve GitHub. I'm currently avoiding all "discussions" altogether because of this feature.

**Comment by leandro21b** [comment]
Ajuda muito! Eu sugiro que também adicione as traduções para `repositórios` e `issues` se possível.

**Comment by Falcowoski** [comment]
Very good!

**Comment by LauraRaellen** [comment]
Acho que é ótimo está função porque facilita o entendimento, mas muitas das vezes a tradução está errada retirando isso ajuda na `compreensão.`

**Comment by werickdasilva** [comment]
Very good

**Comment by yurikerber** [comment]
Muito boa a tradução!!!

**Reply by kevinboolean** [reply]
well, a minha visão deste desafio de sonificação , está focado em realmente extrair um espectro de frequencia da imagem , seja qualquer que seja e emitir o som ...obvio da frequencia obtida com varias tecnicas que podem ser vistas em metodos numericos , IA , ou computação grafica .....legal o tema , boa sorte a todos

**Reply by an0nimo** [reply]
Dude... que viagem é essa, véio‼️
Eles só querem saber a opinião quanto a tradução para o idioma português.... kkkk.

**Comment by Williamhenry11** [comment]
Hello there! 👋

Thank you for starting this discussion to gather feedback on the beta release of Brazilian Portuguese translations for Discussions. Providing feedback is essential to improve and refine features, so your input is greatly appreciated.

Here are some tips on how you can provide valuable feedback:

1. User Experience: Share your overall experience using the Brazilian Portuguese translations. Were you able to navigate and understand the content easily?

2. Accuracy: Comment on the accuracy of the translations. Did you notice any errors or mistranslations? If so, please provide specific examples.

3. Clarity: Evaluate the clarity of the translated content. Were the translations clear and concise, or were there instances where the meaning was unclear?

4. Suggestions for Improvement: If you have any ideas on how to enhance the translation feature or make it more user-friendly, please share them. Constructive suggestions are valuable.

5. User-Friendliness: Consider the ease of using the feature. Did you encounter any difficulties in accessing or using the translations?

6. Performance: Report any issues related to the performance of the translations, such as slow loading times or technical glitches.

7. User Interface: Comment on the user interface of the translation feature. Was it intuitive and easy to use?

By providing specific feedback on these aspects, you can help the developers and the community improve the Brazilian Portuguese translations for Discussions and enhance the overall user experience for Portuguese-speaking users.

Remember that your feedback is valuable in making improvements, so don't hesitate to share your thoughts and observations. It's a collaborative effort that benefits everyone who uses the feature. Thank you for taking the time to contribute! 🙌

Credit: [Invasioned](https://invasioned.com/)

**Comment by FelipeCarvalhoH** [comment]
Very good!

**Comment by PEAL-26** [comment]
very good

**Comment by louiemoreira76** [comment]
nice!

**Comment by an0nimo** [comment]
Very nice and useful.  ![legal](https://github.com/community/community/assets/52868596/632ec326-778c-4f89-8e74-fc3d2ccd14ba)

**Comment by maisondasilva** [comment]
Very good! Great job!

**Comment by iandiasgamerbr0202** [comment]
Thank you for creating this tool, it's helping me a lot.
There is no negative opinion from my side, so, congrats! It's working.

**Comment by livehass** [comment]
Very good!

**Comment by robsonware** [comment]
really nice feature. (in pt-BR: _recurso pica_!)

**Comment by LuPenteado** [comment]
Meu GitHub não aparece em PORTUGUÊS, mesmo estando configurado para usar IDIOMA PORTUGUÊS, não sei porque isso acontece. Eu estou usando Windows 7, no PC. Hoje, 31/05/24 Brasil

**Comment by rogtavares** [comment]
gostei, e muito

**Comment by bielaugustos** [comment]
A tradução funciona muito bem.
Parabéns pela iniciativa!

**Comment by Carlos-Levi** [comment]
Fala pessoal, queria pedir um favor a vocês. No meu perfil existe um **repositório** sobre a linguagem **lua** 

se vocês puderem dar uma star ⭐️, eu agradeço e retribuo !!

Quem quiser seguir, tmj também 🇧🇷

**Comment by IsaqueCar** [comment]
As a native, i can say the translations are kind of good!
it's transalet to a more formal way and it migth be a little boring so use some words like "tá" instead of "está" as an example

**Comment by r0b14** [comment]
I thought this idea was fantastic. I believe there's a lot of room for improvement and for trying to incorporate native English words that aren't so easily translated, because I'm always trying to deduce how we would understand concepts and vocabulary that are already rooted in English.

**Comment by Jeiel0rbit** [comment]
Very good, it gives me an idea of what it's about even if the translations are wrong.

**Comment by alradorbro0001794-cmyk** [comment]
Retxxgxyrrfxgdwwjvhgvyhhjygzgshghvxxnxbhfthsfxxtyfxxfzhygfyhy

**Comment by Lorenz0918** [comment]
oi eu joguei esse jogo e no level 74 da agua ao em vez de ir pra direita voce olha para a esquerda e tem um bloco azul e nao consigo chegar ate  ele e queria saber oque é 
<img width="1365" height="767" alt="Captura de tela 2025-11-10 123206" src="https://github.com/user-attachments/assets/93c39885-be75-42bc-800c-d3351cf3d4b5" />

**Comment by CesarVallada** [comment]
> Hello! 👋 This is a discussion for gathering user feedback on the beta release of Brazilian Portuguese translations for Discussions.
> 
> To learn more about this feature and how to use it, see the [changelog post](https://github.blog/changelog/2021-07-19-machine-translation-for-discussions-content/).
> 
> Please use this discussion to share your experience using the new feature, and any ideas you have on how the experience could be improved.
